### DISCUSSION TITLE
9.25购买的1年专业版，还是免费版，无法升级专业版，麻烦处理下谢谢

### DISCUSSION THREAD

**Discussion by Joah618** [getlantern/lantern-forum-cn#890] [Q&A] [answered]
麻烦处理下付款后无法升级专业版问题
![d5c2a3b5077d499923d73f492f12b43f](https://github.com/user-attachments/assets/0681c525-04ed-4a3e-9f07-4d320012bd13)

**Comment by Joah618** [comment]
@Mike777Khoo

**Comment by MaxMKMax** [comment]
@Joah618 非常抱歉. 好的. 我已经帮您提交上去了. 请您放心! 谢谢您支持蓝灯!

**Comment by Joah618** [comment]
@Mike777Khoo 还没解决？

**Comment by greySkiesMarsh** [comment, marked as answer]
@Joah618 Hello, I'm very sorry for the delay. 

Please use this code to upgrade your account:
M6KXP-VHTTB-Y3TPX-4CH6C-VV994
